core/controllers/cron.py

```diff
@@ -39,14 +42,105 @@
 MAX_JOBS_TO_REPORT_ON = 50
 
 
+def get_stuck_jobs(recency_msecs):
```

Comment:
These functions are shifted from jobs.py to cron.py as they were exclusive to cron

Reply:
Seems to be in the wrong place though. Cron.py contains controllers, which are just meant to be thin interfaces to domain logic.

I'm happy with putting this in a new core/domain/cron_services.py file if you like, but we shouldn't have it in the controllers/ layer.

Reply:
done